From 5529c6ebb8b02a6e5cbc7c7fca8304ba53db7ef3 Mon Sep 17 00:00:00 2001
From: "Tan, Raymond" <raymond.tan@intel.com>
Date: Fri, 17 Apr 2015 13:12:57 +0800
Subject: [PATCH 2/2] spi-pxa2xx: add support up to 4 slaves per master - RTC
 #87333

This patch is adopted from R1.1 to support up to 4 slaves per master
on Quark X1000 SoC platform.

The original author of the changes in R1.1 is
Josef Ahmad <josef.ahmad@intel.com>.

Signed-off-by: Tan, Raymond <raymond.tan@intel.com>
Signed-off-by: Pengyu Ma <pengyu.ma@windriver.com>
---
 drivers/spi/spi-pxa2xx-pci.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/spi/spi-pxa2xx-pci.c b/drivers/spi/spi-pxa2xx-pci.c
index 88b8f91..d35925e 100644
--- a/drivers/spi/spi-pxa2xx-pci.c
+++ b/drivers/spi/spi-pxa2xx-pci.c
@@ -96,7 +96,7 @@ static struct pxa_spi_info spi_info_configs[] = {
 	[PORT_QUARK_X1000] = {
 		.type = QUARK_X1000_SSP,
 		.port_id = -1,
-		.num_chipselect = 1,
+		.num_chipselect = 4,
 		.max_clk_rate = 50000000,
 	},
 };
-- 
1.7.5.4

